src/components/datepicker/js/calendar.js

```diff
@@ -57,7 +57,8 @@
       scope: {
         minDate: '=mdMinDate',
         maxDate: '=mdMaxDate',
-        dateFilter: '=mdDateFilter'
+        dateFilter: '=mdDateFilter',
+        _currentView: '@mdCurrentView'
```

Comment:
I can't understand it either, the only time something interacts with it is when passing it through the template.  My initial guess was because the calendar pane gets transferred from being a child of the datepicker to being directly in the body, but it still happened even after disabling it.